api/sbom.proto

```diff
@@ -144,6 +145,17 @@ message Tool {
     string vendor = 3;
 }
 
+message Lifecycle {
+    string name = 1;
+    optional string description = 2;
+    optional bool enum = 3;
+}
```

Comment:
Perfect, how can I find more design decisions like these? 

SPDX 3.0 uses "Lifecycle"-like object 
https://github.com/spdx/spdx-3-model/blob/27371c9d9b0d2e2c6cbee6275736087c194489ae/model/Core/Vocabularies/LifecycleScopeType.md?plain=1#L3

So it comes to "us" using this the correct naming while both specs uses "Lifecycle" terminology.

Reply:
@jspeed-meyers thank you for your comment, as I'm still onboarding into this project and don't know the full history and prior art regarding the work on `protobom`, My approach is to mostly execute and build while accepting things "as-is".  

I referred to https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf as something I wasn't aware of. It would be best to refer our design doc to important documents like this. 

Do we have more references and resources for SBOM consensuses achieved by the community?

Reply:
So I guess we need to add `documentTypes` sourced from the CISA document into our design doc right?

Reply:
Right, my view is that we should follow the SBOM community, especially when documents with guidance are debated and published.